### PR TITLE
Add basic ee telemetry metrics

### PR DESCRIPTION
**Pull request by jackklpan**
Signed-off-by: Jack Pan <jackpan@infuseai.io>

### CODE REVIEW

**Comment by jackklpan** [comment]
Image Tag: ch14143-e727de71

File: packages/graphql-server/src/ee/app.ts [hunk 1]
```diff
@@ -399,7 +405,7 @@ export const createApp = async (): Promise<{app: Koa, server: ApolloServer, conf
           kcAdminClient.setAccessToken(accessToken);
           getInstanceType = instCache.get;
           getImage = imageCache.get;
-          readOnly = true;
+          readOnly = false;
```
Comment (popcornylu) [inline packages/graphql-server/src/ee/app.ts:408]: 1. Could we remove all readOnly code for both `app.ts` and `ee/app.ts`
2. Remove readOnlyMiddleware code.

**Review by popcornylu — COMMENTED** [review]
Please remove readOnly code if we no longer use it.

**Comment by popcornylu** [comment]
LGTM

**Pull request merged**